test/unit-test/ice_api/ice_api_utest.c

```diff
@@ -1906,14 +2173,86 @@ void test_iceHandleStunPacket_BindingRequest( void )
 /*-----------------------------------------------------------*/
 
 /**
- * @brief Validate ICE Handle Stun Packet functionality.
+ * @brief Validate ICE Handle Stun Packet functionality for Binding Request Type.
  */
-void test_iceHandleStunPacket_IntegrityMismatch( void )
+void test_iceHandleStunPacket_BindingRequest_DeserializationError( void )
```

Comment:
Could you describe the fail part in this STUN message in the comment? Or I missed it?

Reply:
The local Password length is 0, will add the comment for it.